Tests/test_CodonTable.py

```diff
@@ -159,7 +159,7 @@ def test_stop_codons(self):
 
             # R = A or G, so URR = UAA or UGA / TRA = TAA or TGA = stop codons
             if "UAA" in amb_rna.stop_codons and "UGA" in amb_rna.stop_codons \
-               and id != 28:
+               and id not in(28, 32):
```

Comment:
needs a space there, flake8 will likely spot this too.

Reply:
Unfortunately, flake8 didn't flag this.

Reply:
You're right. I wonder if it assumes from the context that ``in(28, 32)`` is calling a function? Logged as https://github.com/PyCQA/pycodestyle/issues/843#issuecomment-496212488 but see also https://github.com/PyCQA/pycodestyle/issues/631

Reply:
Found a better matched issue, https://github.com/PyCQA/pycodestyle/issues/646